js/schema/submit.json

```diff
@@ -0,0 +1,79 @@
+{
+  "type": "object",
+  "additionalProperties" : false,
+  "title": "submit",
+  "properties": {
+    "type": {
+      "type": "string",
+      "enum": ["submit"],
+      "default": "submit",
+      "readonly": true
+    },
+    "title": {
+      "type": "string",
+      "maxLength": 128
+    },
+    "rtl": {
+      "type": "boolean"
+    },
+    "tooltip": {
+      "type": "string",
+      "maxLength": 256
+    },
+    "tag": {
+      "type": "string",
+      "maxLength": 64
+    },
+    "tagVersion": {
+      "type": "string",
+      "maxLength": 64
+    },
+    "style": {
+      "$ref": "style.json"
+    },
+    "alt": {
+      "type": "string",
+      "maxLength": 2000
+    },
+    "click": {
+      "type": "object",
+      "additionalProperties" : false,
+      "properties": {
+        "actions": {
+          "type": "array",
+          "maxItems": 1,
+          "items": [
```

Comment:
@th9061 @gunkoalex 
As a continuance to our conversation, I thought that if we do going in this direction it will be better to create another file named `special_actions.json` that will contain the `submit` action (and any other special action in the future) and reference the submit action here.
This will maintain the current structure and will allow reusing the special actions as well.

Should be something like
on `submit.json`:
```js
"actions": {
          "type": "array",
          "maxItems": 1,
          "items": [
            "$ref": "special_actions.json#/submit"
          ]
        }
```
`special_actions.json`:
```js
{
"submit": {
              "title": "Submit As Text",
              "type": "object",
              "additionalProperties" : false,
              "properties": {
                "type": {
                  "type": "string",
                  "enum": [
                    "submitAsText"
                  ],
                  "default": "submitAsText",
                  "readonly": true
                },
                "submit": {
                  "type": "boolean",
                  "default": false
                }
              },
              "required": ["type", "submit"]
            }
}
```

Pls note that this is a quick sketch and I didn't try to really use it on ajv

Reply:
I will check this and let you know! Thank you for the suggestion!